dobson/config.ini.tmpl

```diff
@@ -0,0 +1,8 @@
+[dobson]
+SlackApiToken = xoxb-00000000000-000000000000-AAAAAAAAAAAAAAAAAAAAAAAA
+DobsonSlackId = UCAAAAAAA
+KnownDevicesFile = dobson/devices.json
```

Comment:
I think you need a sample file for the devices, just to show how the JSON is laid out, but I think putting the devices into a separate file is reasonable, since JSON seems like it works well for storing that kind of data. That being said, maybe it would be easier to just store all of this config in one JSON file? Not sure, but that does at least help to not have 2 separate private config files.

Reply:
Yea, it is annoying having too many config files, but the reason I was thinking of having a standalone file for the devices is so that dobson could write to the file (if we want to be able to tag MAC Addresses from slack). Do you think that alone is worth having a separate file? Or could we still have the use case with it being inside this file?

Reply:
Ah, good point about writing to the file, I think if we want to support adding new users through Slack (which I think would be a pretty nice feature), then having the device list as a separate file would make sense, so I think leaving it like this is fine.